src/main/java/com/epam/ms/controller/NotificationController.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.ms.controller;
+
+import com.epam.ms.repository.domain.Notification;
+import com.epam.ms.service.NotificationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+import static java.util.Objects.nonNull;
+
+/**
+ * REST controller exposes an endpoint to work with notifications
+ * @author Dziyana Bahdanava
+ */
+@RestController
+@RequestMapping("/notifications")
+public class NotificationController {
+
+    @Autowired
+    private NotificationService service;
```

Comment:
Also, they say Field-based injection is not recommended way of injection:
https://www.baeldung.com/inversion-control-and-dependency-injection-in-spring
Read section 7 carefully.
I'd recommend constructor based.

```
@RestController
@RequestMapping("/notifications")
@RequiredArgsConstructor
public class NotificationController {

    private final NotificationService service;
```